From fb13a93b9be748d9fb7fa59f8d25cba3cf9c76eb Mon Sep 17 00:00:00 2001
From: Li Jun <b47624@freescale.com>
Date: Tue, 20 Jan 2015 16:03:35 +0800
Subject: [PATCH 0078/5242] MLK-10086-1 usb: doc: ci-hdrc-imx: update for hsic
 controller

commit  d01a06b372d80632dcc6aefa9ec04c86a8dfebe5 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update for HSIC controller

Signed-off-by: Peter Chen <peter.chen@freescale.com>
(cherry picked from commit c68e2fee7a6e321db9bb56b42853fe90a4e398ee)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/usb/ci-hdrc-usb2.txt       |   10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt b/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
index 9cb0098..7ffc3e9 100644
--- a/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
+++ b/Documentation/devicetree/bindings/usb/ci-hdrc-usb2.txt
@@ -94,6 +94,12 @@ i.mx specific properties
   please do not set it.
 - fsl,anatop: phandle for anatop module, anatop module is only existed
   at imx6 SoC series.
+- pinctrl-names: for names of hsic pin group
+- pinctrl-0: hsic "idle" pin group
+- pinctrl-1: hsic "active" pin group
+- osc-clkgate-delay: the delay between powering up the xtal 24MHz clock
+  and release the clock to the digital logic inside the analog block,
+  0 <= osc-clkgate-delay <= 7.
 
 Examples:
 
@@ -126,6 +132,10 @@ Examples:
 		external-vbus-divider;
 		imx6-usb-charger-detection;
 		fsl,anatop = <&anatop>;
+		pinctrl-names = "idle", "active";
+		pinctrl-0 = <&pinctrl_usbh2_1>;
+		pinctrl-1 = <&pinctrl_usbh2_2>;
+		osc-clkgate-delay = <0x3>;
 		maximum-speed = "full-speed";
 		tpl-support;
 	};
-- 
1.7.9.5

